src/components/accordion/_accordion.scss

```diff
@@ -173,7 +173,7 @@
     position: relative;
     width: 100%;
     margin: 0;
-    transition: background-color $transition--base;
+    transition: background-color motion(standard, productive) 100ms;
```

Comment:
Should we be using a variable here for the duration? (Goes for all the changes)

Reply:
@shixiedesign was saying that we don't currently have a token for these values so I wasn't sure 🤔

Reply:
@shixiedesign is there a reason why we don't have variables for these? If we don't, shouldn't we create them? I just want to ensure transitions are consistent throughout the library. Doesn't necessarily need to be implemented in this PR, but definitely something we should add

Reply:
right we should add variables. Currently we only have `$transition--base` and `$transition--expansion`, which is 100ms and 150ms. I wanna go in and add a few more. Can we wait for me to add a few more values, and then make a batch update to the duration values?

Reply:
Sure! That could work. When should I follow up for the new values?

Reply:
I'll make a PR for it soon!

Reply:
I meant to say update duration values to tokens above.